docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -509,12 +511,17 @@ If you're upgrading to a new version, the upgrade process depends on the deploym
 - [Upgrading&#8212;Terraform Template](#upgradingterraform-template)
 - [Upgrading&#8212;gcloud CLI](#upgradinggcloud-cli)
 
-### Upgrading&#8212;Terraform Template
+#### Upgrading&#8212;Terraform Template
 
 If you deployed using the Terraform template, all you need to do to upgrade is update your deployment with the new `{OPERATOR_IMAGE}` that you received in the upgrade notification.
```

Comment:
I know this is not your change, but we need to change this. It should say:
If you deployed using the Terraform template, please compare the latest version of the template with the one you used to do the deploy.